onnxruntime/test/contrib_ops/matmul_integer_to_float_test.cc

```diff
@@ -82,28 +82,32 @@ void TestMatMulIntegerToFloat(const std::vector<int64_t>& A_dims,
   }
 
   if (has_bias) {
-    test.AddInput<float>("bias", {B_dims.back()}, Bias);
+    test.AddInput<OType>("bias", {B_dims.back()}, Bias);
   } else {
-    test.AddOptionalInputEdge<float>();
+    test.AddOptionalInputEdge<OType>();
   }
 
   test.AddReferenceOutputs(reference_model);
 #if defined(USE_DML)
-  test.SetOutputRelErr("Y", 2e-2f);
+  if constexpr (std::is_same_v<OType, float>) {
+    test.SetOutputRelErr("Y", 2e-2f);
+  } else {
+    test.SetOutputRelErr("Y", 0.3f);
+  }
 #else
   test.SetOutputRelErr("Y", 1e-4f);
 #endif
 
   test.Run();
 }
 
-template <typename IType, typename WType, bool HasZeroPoint, bool HasBias>
+template <typename IType, typename WType, typename OType, bool HasZeroPoint, bool HasBias>
 void RunMatMulIntegerToFloatTest(const string& model_path) {
   std::vector<int64_t> A_dims{4, 128};
   std::vector<int64_t> B_dims{128, 128};
   std::vector<int64_t> Y_dims{4, 128};
 
-  TestMatMulIntegerToFloat<IType, WType>(A_dims,
+  TestMatMulIntegerToFloat<IType, WType, OType>(A_dims,
```

Comment:
>A_dims, [](http://example.com/codeflow?start=48&length=7)

[nit] Should realign the indent now. In VS, this is Ctrl+K, Ctrl+F.

Note you don't have to worry about this in the DML EP and WinML code because they always consistently indent the same amount regardless of function name length (e.g. below) and they wrap parameters onto a new line for easy scannability, but other parts of the ORT follow TensorFlow's style.